packages/fxa-content-server/server/lib/beta-settings.js

```diff
@@ -108,6 +108,11 @@ function modifyProxyRes(proxyRes, req, res) {
       });
       res.send(new Buffer.from(html));
     } else {
+      // remove transfer-encoding header, a Content-Length header will be added
+      // automatically, and these two headers are incompatible. If this is not
+      // set any fetch request will fail with:
+      //  - Parse Error: Content-Length can't be present with Transfer-Encoding
```

Comment:
I don't think it should matter if it is a front end or back end request. It was working on the front end, but per the spec this is actually an invalid state and really shouldn't be accepted. 

If I recall correctly something downstream was settings the content-length so putting a conditional check here wouldn't work out. I can check again though if you think it's actually problematic.

Reply:
Ah I see, I just assumed since I didn't see any errors when using it on the front-end.

> The issue is that 'chunked' proved to be incompatible. I maybe should check to see if it's `gzipped, chunked` and preserve gzip if that is present. 🤔

🤔 What're the implications if we don't check for it?